ovos_adapt/intent.py

```
@@ -15,9 +15,7 @@
 
 __author__ = 'seanfitz'
 
-import itertools
-
-CLIENT_ENTITY_NAME = 'Client'
+from ovos_workshop.intent import Intent, IntentBuilder
```

Ruff

`ovos_workshop.intent.IntentBuilder` imported but unused

Remove unused import: `ovos_workshop.intent.IntentBuilder`

(F401)